src/testCommon/java/uk/gov/hmcts/darts/test/common/data/MediaRequestTestData.java

```diff
@@ -30,6 +34,50 @@ public static MediaRequestEntity minimalRequestData() {
         return mediaRequest;
     }
 
+    public static MediaRequestEntity createCurrentMediaRequest(HearingEntity hearingEntity, UserAccountEntity requestor,
+                                                        OffsetDateTime startTime, OffsetDateTime endTime,
+                                                        AudioRequestType audioRequestType, MediaRequestStatus status) {
+
+        return createCurrentMediaRequest(hearingEntity, requestor, requestor, startTime, endTime, audioRequestType, status, OffsetDateTime.now());
+    }
+
+    public static MediaRequestEntity createCurrentMediaRequest(HearingEntity hearingEntity, UserAccountEntity owner, UserAccountEntity requestor,
+                                                        OffsetDateTime startTime, OffsetDateTime endTime,
+                                                        AudioRequestType audioRequestType, MediaRequestStatus status, OffsetDateTime requestedDate) {
+        MediaRequestEntity mediaRequestEntity = someMinimalRequestData();
+        mediaRequestEntity.setHearing(hearingEntity);
+        mediaRequestEntity.setRequestor(requestor);
+        mediaRequestEntity.setCurrentOwner(owner);
+        mediaRequestEntity.setStatus(status);
+        mediaRequestEntity.setRequestType(audioRequestType);
+        mediaRequestEntity.setAttempts(0);
+        mediaRequestEntity.setStartTime(startTime);
+        mediaRequestEntity.setEndTime(endTime);
+        mediaRequestEntity.setCreatedBy(requestor);
+        mediaRequestEntity.setLastModifiedBy(requestor);
+        mediaRequestEntity.setCreatedDateTime(requestedDate);
+
+        return mediaRequestEntity;
+    }
+
+    public static MediaRequestEntity createCompletedMediaRequest(HearingEntity hearingEntity, UserAccountEntity requestor,
+                                                          OffsetDateTime startTime, OffsetDateTime endTime,
+                                                          AudioRequestType audioRequestType) {
+        MediaRequestEntity mediaRequestEntity = someMinimalRequestData();
```

Comment:
This could return createCurrentMediaRequest(HearingEntity hearingEntity, UserAccountEntity owner, UserAccountEntity requestor, OffsetDateTime startTime, OffsetDateTime endTime, AudioRequestType audioRequestType, MediaRequestStatus status, OffsetDateTime requestedDate) passing MediaRequestStatus.COMPLETED ?

Reply:
Good shout. Having looked at it, I have now removed this overloaded method because it's overly specific to only a single test. The caller can just invoke `createCurrentMediaRequest()` as you say.